packages/plugin-e2e/src/models/VariablePage.ts

```diff
@@ -0,0 +1,32 @@
+import { Expect } from '@playwright/test';
+import { PluginTestCtx } from '../types';
+import { GrafanaPage } from './GrafanaPage';
+import { VariableEditPage } from './VariableEditPage';
+
+export class VariablePage extends GrafanaPage {
+  constructor(ctx: PluginTestCtx, expect: Expect<any>) {
+    super(ctx, expect);
+  }
+
+  async goto() {
+    //TODO: use selector instead
```

Comment:
can you open an issue for this ?

Reply:
Yeah good shout. Task created in the epic!